frameworks/preact/preview/render.tsx

```diff
@@ -0,0 +1,10 @@
+import { render as preactRender, ComponentType } from 'preact';
+
+export function render<P>(Renderer: ComponentType<P>, props: P) {
```

Comment:
FYI this can be in `preview/index.tsx`, the only reason to have it separate for React is that its implementation differs based on React versions.